aggregator/src/datastore.rs

```diff
@@ -2149,6 +2149,92 @@ WITH updated as (
         WHERE
             -- Constraint for tasks table in FROM position
             tasks.id = collect_jobs.task_id
+            -- Only return
+            AND tasks.query_type ? 'TimeInterval'
```

Comment:
Why do we use the `?` operator here? If I'm reading [the docs](https://www.postgresql.org/docs/14/functions-json.html) right, that will test whether `TimeInterval` appears as a "top-level key or array element", but don't we want to do simple string comparison between `query_type` and `TimeInterval`?

Reply:
I tried this out, and there is no `jsonb = text` operator defined.

```
postgres=# SELECT '"TimeInterval"'::jsonb = 'TimeInterval'::text;
ERROR:  operator does not exist: jsonb = text
```

While that doc page doesn't mention it, using `?` on a JSON value that's a string is equivalent to unwrapping that string and comparing it to the `text` operand.

Reply:
Ah, here we go, this is mentioned in an example on this page: https://www.postgresql.org/docs/14/datatype-json.html#JSON-CONTAINMENT

```
-- A string is considered to exist if it matches a primitive JSON string:
SELECT '"foo"'::jsonb ? 'foo';
```

I submitted a comment to the pgsql-docs mailing list about fleshing this out.